taxonium_web_client/src/components/MutationHoverTip.jsx

```diff
@@ -0,0 +1,70 @@
+import { useMemo } from "react";
+
+
+const MutationHoverTip = ({ hoverInfo, hoverDetails, colorHook, colorBy, config, reference }) => {
+  if (!hoverInfo || !reference) {
+    return null;
+  }
+  const hoveredMutation = hoverInfo.object;
+
+  if (!hoveredMutation || !hoveredMutation.m) {
+    return null;
+  }
+  const posKey = hoveredMutation.m.gene + ':' + hoveredMutation.m.residue_pos;
+  if (hoveredMutation.m.new_residue == reference['aa'][posKey]) {
+    return null;
+  }
+
+
+  return (
+    <div
+      className="bg-gray-100 p-3 opacity-90 text-sm"
+      style={{
+        position: "absolute",
+        zIndex: 0,
+        pointerEvents: "none",
+        left: hoverInfo.x,
+        top: hoverInfo.y,
+      }}
+    >
+      <h2 className="font-bold whitespace-pre-wrap">
+      </h2>
+      {/* {colorBy.colorByField === "genotype" && (
```

Comment:
I think I like this change to not have these colours in the tooltip. We can lose the comment once we're confident about that.

Reply:
Yeah, It was pretty difficult to read when the colors were on the lighter side

Reply:
Nice